examples/Test-Summation/Jenkinsfile

```diff
@@ -15,7 +15,7 @@ node {
         }
 
         stage('deploy'){
-            legion.deploy()
+            legion.deploy(timeout: 15)
```

Comment:
Maybe this should be liveness and readiness timeouts instead of a single timeout?

Reply:
added separate property for readiness probe